components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/multiColumnRowView/MultiColumnRowView.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.hmrc.components.compose.molecule.multiColumnRowView
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+fun MultiColumnRowView(
```

Comment:
We know our MultiColumnRow should only have up to 3 columns, so can we restrict the usage a bit further to only allow at max 3 items? We also know the one closest to the right edge of the screen should always be right aligned, so we can build in some rules to the component to enforce that, so the consumer doesn't need to define it each time they use it. It would also be good if the consumer didn't need to define the weight of each column item each time, because the component should be able to determine that itself based on the number of columns.

See here for visual examples: 
<img width="397" alt="Screenshot 2023-04-25 at 12 30 28" src="https://user-images.githubusercontent.com/7998113/234263571-e9c962fa-45e0-4414-8469-a7f32e7cbcad.png">

https://hmrc.invisionapp.com/console/share/EXOVJCPYDPN/332983439